test/buglist_test_vectors.md

```diff
@@ -0,0 +1,41 @@
+# NestedArrayFunctionCallDecoder
+
+## buggy
+
+function f() returns (uint[2][2]) { }
+
+--
+
+function f() returns (uint[2][2] a) { }
+
+--
+
+function f() returns (uint x, uint[200][2] a) { }
+
+--
+
+function f() returns (uint[200][2] a, uint x) { }
+
+--
+
+function f() returns (uint[200][2] a, uint x);
+
```

Comment:
Add a test that has newlines after `return` and a test that uses a constant or expression for array size: `uint[2*2][2]` `uint[X][2]`

Reply:
Done.